stable/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.1.6
+version: 6.1.7
```

Comment:
According to semver versioning, this version should be bumped to `6.2.0`

Other than that, the PR looks good.

Reply:
Corrected in bd48a26fb1f4c1e208efc4b486b985b18c197156